app/src/main/res/values/strings.xml

```diff
@@ -15,4 +15,22 @@
     <string name="bins_pref_label">Bins</string>
     <string name="num_channels_pref_label">Number Channels</string>
 
+    <string name="launcher_cbs_text_view_title">Settings</string>
```

Comment:
Wrong use of string resources. Just name this string "settings". Same everywhere else

Reply:
@iamareebjamal Shouldn't I use string resources for the Settings title?